app/src/main/kotlin/com/wire/android/ui/home/conversationslist/model/Call.kt

```diff
@@ -3,16 +3,29 @@ package com.wire.android.ui.home.conversationslist.model
 import androidx.annotation.DrawableRes
 import com.wire.android.R
 
-data class Call(val callInfo: CallInfo, val conversation: Conversation)
+data class Call(
+    val callInfo: CallInfo,
+    override val conversationType: ConversationType
+) : Conversation()
 
-data class CallTime(val date: String, val time: String) {
+data class CallTime(
+    val date: String,
+    val time: String
```

Comment:
Wouldn't be better to have `time` as a `Long` timestamp? 🤔

Reply:
Good suggestion but I'll leave as it is and add a comment instead of changing itand wait for the Kalium so that I can format it to correct String by then